lib/utils.js

```diff
@@ -307,8 +307,16 @@ function prettifyObject ({
 
   let result = ''
 
-  const keysToIterate = objectKeys.filter(k => keysToIgnore.includes(k) === false)
-  for (var i = 0; i < objectKeys.length; i += 1) {
+  function safeObjectKeys () {
+    const safePrims = objectKeys.filter(k => k !== 'null')
+    const strArr = keysToIgnore.join(',').split(',')
+    const sanitized = safePrims.filter(k => strArr.includes(k) === false)
+    return sanitized
+  }
+
+  const keysToIterate = safeObjectKeys()
+  var i = 0
+  for (i; i < objectKeys.length; i += 1) {
```

Comment:
I couldn't help but notice that this code is again iterating `objectKeys` array but using the index on `keysToIterate` variable. Shouldn't this be :

```suggestion
  for (i; i < keysToIterate.length; i += 1) {
```

Reply:
Thanks for catching this Rohit, this was an unintended reverse during branch changing